src/cli/argument-parser.js

```diff
@@ -85,6 +85,51 @@ export default class CLIArgumentParser {
         return '\n' + wordWrap(DESCRIPTION, 2, getViewPortWidth(process.stdout));
     }
 
+    static _parseBrowserArg (browserArg) {
+        var browser  = '';
+        var browsers = [];
+
+        var isInQuotedPart = false;
+        var quoteChar      = '';
+
+        for (var i = 0; i < browserArg.length; i++) {
+            var currentChar = browserArg[i];
+
+            switch (currentChar) {
+            case ',':
+                if (isInQuotedPart)
+                    browser += currentChar;
+                else {
+                    browsers.push(browser);
+                    browser = '';
+                }
+                break;
+
+            case '"':
+            case '\'':
+                if (!isInQuotedPart) {
+                    isInQuotedPart = true;
+                    quoteChar      = currentChar;
+                }
+                else if (quoteChar === currentChar) {
+                    isInQuotedPart = false;
+                    quoteChar      = '';
```

Comment:
Seems like `isInQuotedPart` var is redundant. We can set `quoteChar` to null when we are not in quote (this is actually what happens currently) and use it as flag.